### ISSUE TITLE
Slash appears when switching 

### ISSUE THREAD

**Issue by qianniancn** [fyne-io/fyne#2442]
### Describe the bug:

Slashes appear when switching

###  The sample code is used :
https://github.com/fyne-io/examples

### Screenshots:

![GIF 2021-9-7 20-20-40](https://user-images.githubusercontent.com/5478237/132343913-a6041bda-0c71-4d2d-8601-a4d1424e0f9e.gif)

### Device (please complete the following information):
 - **OS:** windows10
 - **Version:** 10
 - **Go version:** 1.17
 - **Fyne version:** 2.0.4

**Comment by andydotxyz** [comment]
Please provide more information about your graphics hardware and driver, this has never been observed before.

**Comment by qianniancn** [comment]
@andydotxyz The Intel Iris Xe was used，This small problem does not affect the use

**Comment by andydotxyz** [comment]
Not sure why this was closed, can you comment @qianniancn?

**Comment by qianniancn** [comment]
@andydotxyz I don't think this issue will affect usage because there are no slashes when switching options in fyne_demo
